docs/sources/alerting/silences/remove-silence.md

```diff
@@ -0,0 +1,15 @@
++++
+title = "Remove silences"
+description = "Remove alerting silences"
+keywords = ["grafana", "alerting", "silence", "mute"]
+weight = 451
+aliases = ["/docs/grafana/latest/alerting/unified-alerting/silences/"]
++++
+
+# Remove silences
+
```

Comment:
no intro or before you begin?

Reply:
I am not sure what information will be useful here. Suggestions?